### PR TITLE
meta-mender-atmel: refresh for kirkstone

### PR DESCRIPTION
**Pull request by TheYoctoJester**

### CODE REVIEW

**Comment by TheYoctoJester** [comment]
Hi @texierp, can you maybe check if this looks right - eventually even give it a test spin? Thanks!

**Comment by TheMeaningfulEngineer** [comment]
Thanks!
Only briefly skimmed, if question doesn't make sense it's probably me.
You started a migration to `kas` with this one?
Wouldn't that have an influence of [the readme](https://github.com/mendersoftware/meta-mender-community/tree/kirkstone/meta-mender-atmel)

**Review by kacf — APPROVED** [review]

File: meta-mender-atmel/recipes-bsp/u-boot/u-boot-mender-atmel.inc [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
+SRC_URI:append:sama5d3-xplained = "\
+	file://0001-ARM-Add-Mender-support-for-sama5d3_xplained_nandflas.patch \
+"
+
 MENDER_UBOOT_AUTO_CONFIGURE = "0"
 BOOTENV_SIZE = "0x1000"
+BOOTENV_SIZE:sama5d3-xplained = "0x20000"
+
+MENDER_UBOOT_ENV_STORAGE_DEVICE_OFFSET_1:sama5d3-xplained = "0x140000"
+MENDER_UBOOT_ENV_STORAGE_DEVICE_OFFSET_2:sama5d3-xplained = "0x100000"
```
[the remark this answers is not in the document]
Comment (TheYoctoJester) [reply meta-mender-atmel/recipes-bsp/u-boot/u-boot-mender-atmel.inc:19]: Expected, to match the u-boot defconfig of the board.

[the remark this answers is not in the document]
Comment (TheYoctoJester) [reply meta-mender-atmel/recipes-bsp/u-boot/u-boot-mender-atmel.inc:19]: It actually matters, the other way round does not match the `u-boot` configuration.

**Pull request merged**